services/tone_analyzer/v3-beta.js

```diff
@@ -62,32 +62,49 @@ module.exports = function (RED) {
         version_date: '2016-02-11'
       });
 
-      var options = {
+      var hasJSONmethod = (typeof msg.payload.toJSON === 'function') ;
+      var isBuffer = false;
+      if (hasJSONmethod==true)
+      {
+        if (msg.payload.toJSON().type == 'Buffer')
+            isBuffer=true;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'isBuffer'.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2393366918/issues/source?bid=3225115&fileBranchId=3230140#l70)